test/ut/nas/test_experiment.py

```diff
@@ -112,6 +113,46 @@ def test_multitrial_experiment(pytestconfig):
     assert isinstance(exp.export_top_models()[0], dict)
     exp.stop()
 
+def test_multitrial_experiment_resume_view(pytestconfig):
```

Comment:
Please put this in algo test. I'd like to keep `test_experiment` as minimal as possible.

Reply:
but logically it is the test of experiment, putting in algo would make it hard to find this test. why you want to keep `test_experiment` as minimal as possible?

Reply:
To avoid UT from taking too long to finish.

There is a `test_multitrial` in test/algo.

Reply:
yeah, I can move the test to "test_multitrial", but it is nothing about "algo", you may come up with a good name.